lib/search/document.js

```diff
@@ -82,21 +135,17 @@ Document.prototype.addField = function(name) {
 };
 
 /**
- * Delete this document.
- *
- * @resource [Documents: delete API Documentation]{@link https://cloud.google.com/search/reference/rest/v1/projects/indexes/documents/delete}
- *
- * @param {function=} callback - The callback function.
- * @param {?error} callback.err - An error returned while making this request
- * @param {object} callback.apiResponse - The full API response.
+ * Create a document.
  *
  * @example
- * document.delete(function(err, apiResponse) {});
+ * document.create(function(err, document, apiResponse) {
+ *   if (!err) {
+ *     // The document was created successfully.
+ *   }
+ * });
  */
-Document.prototype.delete = function(callback) {
-  this.makeReq_('DELETE', '', null, null, function(err, resp) {
-    (callback || util.noop)(err, resp);
-  });
+Document.prototype.create = function(callback) {
```

Comment:
Actually that's just a list of methods to implement. It might not be a bad idea to accept functions there as well, though I wouldn't prefer the distance/variable placement of the function definitions.